packages/raystack/calendar/calendar.tsx

```diff
@@ -1,21 +1,36 @@
-import { DayPicker, DayPickerProps, DropdownProps } from "react-day-picker";
-import { cva } from "class-variance-authority";
+import {
+  dateLib,
+  DayPicker,
+  DayPickerProps,
+  DropdownProps,
+} from 'react-day-picker';
+import { cva } from 'class-variance-authority';
 import {
   ChevronRightIcon,
   ChevronLeftIcon,
   ChevronUpIcon,
   ChevronDownIcon,
-} from "@radix-ui/react-icons";
-import styles from "./calendar.module.css";
-import { Select } from "~/select";
-import { ChangeEvent, useEffect, useState } from "react";
-import { Flex } from "~/flex/flex";
+} from '@radix-ui/react-icons';
+import styles from './calendar.module.css';
```

Comment:
Let's keep the css in the end and third party on the top of the imports?

We will automate import sorting later on.